src/shared/components/palette/document/write-document-field.component.ts

```diff
@@ -198,9 +210,17 @@ export class WriteDocumentFieldComponent extends AbstractFieldWriteComponent imp
 
   getUploadedFileName() {
     if (this.uploadedDocument) {
-      return this.uploadedDocument.get('document_filename').value;
+      return this.uploadedDocument.get(WriteDocumentFieldDefinitios.DOCUMENT_FILENAME).value;
     } else {
       return undefined;
     }
   }
 }
+
+class WriteDocumentFieldDefinitios {
+  static readonly DOCUMENT_URL = 'document_url';
+  static readonly DOCUMENT_BINARY_URL = 'document_binary_url';
+  static readonly DOCUMENT_FILENAME = 'document_filename';
+  static readonly UPLOAD_ERROR = 'File required';
```

Comment:
Would it be better if we change to `UPLOAD_ERROR_FILE_REQUIRED`

Reply:
DONE :)